helm/hpcc/templates/sasha.yaml

```diff
@@ -52,6 +52,7 @@ kind: ConfigMap
 {{- if and (not $sasha.storage.existingClaim) (not $sasha.storage.plane) }}
```

Comment:
I think sasha should have an [optional] storagePlane, that must exist.
If ephemeral it a PVC will have been created by addPVCsFromPlanes

Reply:
I think I will leave sasha to you - it seems to behave differently from the other examples.